plays/dc1/k8s-bootstrap.yml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
Lek iki, prasaku kok mending stage `Ensure python is present` iku langsung jadikan role `common` jadi di playbook bersih, ngga ada task tambahan

```
---
- name: Setup kubernetes bootstrap
  hosts: all
  become: yes
  vars:
    wawa: val
    qaaqa: val
    sad: [ ls1, ls2, ls3 ]

  roles:
    - common
    - role2
    - role3
```

tujuane cek modular ae, bersih playbook e, dan konfigurasi untuk ensure ensure, pre dependency iku terkontrol nde satu file, role

Reply:
okee, will be added as role